engine/lite/components/request-response-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/requestresponse/LiteRequestResponseFunctionalTest.scala

```diff
@@ -180,6 +180,8 @@ class LiteRequestResponseFunctionalTest extends AnyFunSuite with Matchers with E
       //Primitive integer validations
       (config(fromInt(1), schemaInt, schemaInt), Valid(fromInt(1))),
       (conf(schemaInt, 1), Valid(fromInt(1))),
+      //Everit cannot handle Bigdecimal as integer, even it represents one...
```

Comment:
How everit behave then? Some exception is thrown? Maybe we could extract this test from table to separate test showing that exception is thrown?

Reply:
Everit throws an exception. At the end of this file is a dedicated test for errors. We can move it there and add comment.